CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ else (CMAKE_VERSION VERSION_LESS 3.0.0)
 	project(libpointmatcher VERSION ${POINTMATCHER_PROJECT_VERSION})
 endif (CMAKE_VERSION VERSION_LESS 3.0.0)
 
+set(LIB_NAME pointmatcher) # same than PROJECT_NAME, w/o the "lib" prefix
+
+include(GNUInstallDirs) # populate CMAKE_INSTALL_{LIB,BIN}DIR
```

Comment:
Can we move all the includes to a single block? Or at least most of them.

Also, would it make sense to use include_directories?

Reply:
Done!